packages/openapi/src/lib.ts

```diff
@@ -22,7 +22,27 @@ export const $lib = createTypeSpecLibrary({
         default: paramMessage`${"property"}: ${"value"} is not a valid URL.`,
       },
     },
+    "duplicate-tag": {
+      severity: "error",
+      messages: {
+        default: paramMessage`"Metadata for tag '${"tagName"}' was specified twice."`,
+      },
+    },
+    "no-service-found": {
+      severity: "warning",
```

Comment:
```suggestion
      severity: "error",
```
this is an error, the tag has not purpose if not on a service namespace